assets/scripts/aircraft.js

```diff
@@ -787,13 +858,16 @@ var Aircraft=Fiber.extend(function() {
       return ['ok', 'turn ' + direction + 'heading ' + heading_to_string(this.requested.heading), ''];
     },
     runAltitude: function(data) {
-      var split     = data.split(" ");
+      if(data[0] == "\u2B61" || data[0] == "\u2B63") {  //shortKey 'v' or '^' in use
```

Comment:
Sure. Only reason I did it at the beginning of each heading/altitude/speed/etc's function instead of runCommand() was because of the left/right turns associated with the shortKeys for heading.... in that case, the arrow needs to be passed in order for runHeading() to know the direction of the turn. I then figured consistency was better than not, and stripped it off at the beginning of each command instead of when building the list of commands. I can definitely change it though if needed.